scripts/2.TCGA-MLexample-covariates.py

```diff
@@ -221,7 +218,7 @@ def select_feature_set_columns(X, feature_set):
     cv_results_df = cv_results_df.append(df)
 
 
-# In[18]:
+# In[17]:
 
 # Cross-validated performance heatmap
```

Comment:
I like the decision to plot model by alpha. I don't think a heatmap is the right visualization since model is categorical. I'd say alpha on x-axis and performance on y-axis, with lines colored by model. (Preferably same color as ROC curve).

This is likely scope creep, so feel free to save this for a future PR.